client/model.go

```diff
@@ -393,7 +393,7 @@ type PolicyUpdatePayload struct {
 
 type EnvironmentSchedulingExpression struct {
 	Cron    string `json:"cron"`
-	Enabled bool   `json:"enabled"`
+	Enabled bool   `json:"enabled,omitempty"`
```

Comment:
I think the `omitempty` should be on `cron` and not on `enabled`.

A simple test in `model_test.go` could test this behavior